src/models.py

```diff
@@ -554,6 +569,8 @@ def build_decoder(task, d_inp, vocab, embedder, args):
 class MultiTaskModel(nn.Module):
```

Comment:
Optional, but it seems like some of the other functions in this file, like `build_task_specific_modules`, might be better implemented as class methods, since they modify this object?